ssg/build_yaml.py

```diff
@@ -570,6 +570,52 @@ def from_yaml(yaml_file, env_yaml=None):
         rule.validate_references(yaml_file)
         return rule
 
+    def normalize(self, product):
+        self.make_items_product_specific(product)
```

Comment:
The method `normalize` doesn't do anything additional on top of `make_items_product_specific`. Please remove one of them.